android/sdl_android/src/main/java/com/smartdevicelink/managers/video/VideoStreamManager.java

```diff
@@ -397,7 +527,7 @@ public void resumeStreaming() {
     @Override
     @RestrictTo(RestrictTo.Scope.LIBRARY)
     public void dispose() {
-        stopStreaming();
+        stopStreaming(false);
```

Comment:
If the proposed changes are made for the stopStreaming methods, this change can be reverted